src/base/system_parameter.c

```diff
@@ -3635,7 +3635,7 @@ static SYSPRM_PARAM prm_Def[] = {
    (DUP_PRM_FUNC) NULL},
   {PRM_ID_LIST_MAX_QUERY_CACHE_PAGES,
    PRM_NAME_LIST_MAX_QUERY_CACHE_PAGES,
-   (PRM_FOR_SERVER),
+   (PRM_FOR_SERVER | PRM_FORCE_SERVER),
```

Comment:
PRM_ID_LIST_MAX_QUERY_CACHE_ENTRIES and PRM_ID_LIST_MAX_QUERY_CACHE_PAGES should apply PRM_HIDDEN same like PRM_ID_LIST_QUERY_CACHE_MODE?